pandas/core/generic.py

```diff
@@ -6110,13 +6110,13 @@ def replace(self, to_replace=None, value=None, inplace=False, limit=None,
               <http://docs.scipy.org/doc/scipy/reference/interpolate.html#univariate-interpolation>`__
               and `tutorial documentation
               <http://docs.scipy.org/doc/scipy/reference/tutorial/interpolate.html>`__
-            * 'from_derivatives' refers to BPoly.from_derivatives which
+            * 'from_derivatives' refers to ``BPoly.from_derivatives`` which
```

Comment:
I think we just use double backticks for literals so this is probably better served with just single back ticks. @datapythonista

Reply:
A few lines above, I see `scipy.interpolate.interp1d` wrapped in double backticks. I figured we would want to do the same for `scipy.interpolate.BPoly.from_derivatives`. Do you want me to change `scipy.interpolate.interp1d` to single backticks?

P.S. Should these additionally be hyperlinks to the actual scipy docs? Or do the scipy docs reorganize enough that it wouldn't be stable?

Reply:
Afaik double backticks are mainly used for code, as they are rendered in monospace. While single backticks don't have a specific meaning, as they can be customized. So, for my understanding, we should use single backticks in this case. But I think in many cases we're using double backticks for similar things.

About the links, they are built automatically for other parts of the documentation, by prefixing `:class:`, `:meth:`, `:ref:`... But I'm not sure if for external packages we've got it set up.

May be @TomAugspurger knows. Otherwise, I'll research and see what's the best we can do.

Reply:
Links to scipy should work, since they're in our intersphinx mapping: https://github.com/pandas-dev/pandas/blob/2156431c3a673fbfb720983474d3cb4edc92c332/doc/source/conf.py#L366-L374 so using `:meth:` is probably best.